Enable Process metric collection info added (#3468)
SumoLogic/sumologic-documentation: docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md

- `Messages`
- `yum.log`

:::note 

> By default, the path for Linux log files required for all the distros are pre-populated in the UI. Not all of the files might be available on your Linux distribution. Optionally, you can remove unwanted file paths from the list. OpenTelemetry collection will still work properly even if not all of the files are present on your system. If in doubt, you can leave the default file paths values.

Reviewer: ```suggestion
By default, the path for Linux log files required for all distros are pre-populated in the UI. Not all of the files might be available on your Linux distribution. Optionally, you can remove unwanted file paths from the list. Our OpenTelemetry collection will still work properly, even if not all of the files are present on your system. If in doubt, we recommend leaving the default file path values.
```

:::

#### Enable process metric collection (Optional)

By default, the collector will not send process metrics to Sumo Logic. This is because the number of processes running on a host can be very large, which would result in a significant increase in Data Points per Minute (DPM).

Click the **Enable process metric collection** checkbox to collect (process level metric)[https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md].

> - **Name of process**. Add the list of process names.

Reviewer: ```suggestion
- **Name of process**. Adds the list of process names.
```

> - **Include/Exclude the above pattern**. Signifies if you want to exclude or include the metrics for the processes listed previously.

Reviewer: ```suggestion
- **Include/Exclude the above pattern**. Includes or excludes the metrics for the processes listed previously.
```

- **Match type for process name**. Select if the process name given should be considered for a strict match with the host machine processes or if it should be considered as regex when matching.

> <img src='img/integrations/hosts-operating-systems/process-metric-collection.png' alt="process-metric-collection" width="400"/>

Reviewer 2: The image is broken because the tagging doesn't include useBaseUrl. For example:
 <img src={useBaseUrl('img/<your-image-file-path>.png')} alt="<your image description>" style={{border: '1px solid black'}} width="<insert-pixel-number>" />

Reviewer: fixed

:::note 
If the process list needs to be edited in the future, you can edit it manually in the OTEL config yaml by adding/removing in the names list under process scrapper.
```sh
process:
  include:
    names: [ <process name1>, <process name2> ... ]
    match_type: <strict|regexp>
```
:::

> <img src='img/integrations/hosts-operating-systems/Linux-YAML.png' alt="Linux-YAML" width="400"/>

Reviewer 2: The image is broken because the tagging doesn't include useBaseUrl. For example:
 <img src={useBaseUrl('img/<your-image-file-path>.png')} alt="<your image description>" style={{border: '1px solid black'}} width="<insert-pixel-number>" />

Reviewer: fixed

> Click on the **Download YAML File** button to get the yaml file.<br/><img src='img/integrations/hosts-operating-systems/Linux-YAML.png' alt="Linux-YAML" width="600"/>

Reviewer 2: This is the same image that appears almost immediately above. Did you mean to call it twice?

Also, the image is broken because the tagging doesn't include useBaseUrl. For example:
 <img src={useBaseUrl('img/<your-image-file-path>.png')} alt="<your image description>" style={{border: '1px solid black'}} width="<insert-pixel-number>" />

Reviewer: fixed
